Purchases/Support/ManageSubscriptionsHelper.swift

```diff
@@ -123,18 +126,20 @@ private extension ManageSubscriptionsHelper {
                   return .failure(ErrorUtils.storeProblemError(withMessage: "Failed to get UIWindowScene"))
         }
 
-        do {
-
 #if os(iOS)
-            try await AppStore.showManageSubscriptions(in: windowScene)
+            _ = Task.init {
```

Comment:
Honestly I think there isn't much that we can do, I think this is fine. I would leave a comment explaining why this is like this instead of just `await`.

Reply:
great idea

Reply:
updated